uber/decorators.py

```diff
@@ -1,28 +1,30 @@
 from uber.common import *
 
+
 def log_pageview(func):
     @wraps(func)
-    def with_check(*args,**kwargs):
-        with Session() as session:
+    def with_check(*args, **kwargs):
+        with sa.Session() as session:
```

Comment:
We now have a global ``sa`` module which is an alias for ``uber`` itself - ``sa`` is a standard abbreviation for ``SQLAlchemy`` and we use this to avoid circular dependency issues with files such as this one which are imported before ``models.py`` but which wish to use things which are defined there.